app/assets/stylesheets/components/home_page_header.scss

```diff
@@ -1,25 +1,24 @@
-header {
-    padding-bottom: 60px;
+.hero {
+  text-align: center;
+  width: 100%;
 
-  h1 {
-    margin-bottom: 30px;
+  header {
+    padding-bottom: 3.75rem;
```

Comment:
admittedly I've been using this PR to experiment with using responsive values. Can we leave it with `rems` to see if it works ok please?

If it becomes a trap I promise I'll fix them 😛